playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart

```diff
@@ -147,7 +154,12 @@ class ExamplesLoader {
     Example example;
     try {
       example = await loader.future;
-    } on Exception {
+    } on MultipleExceptions catch (ex) {
+      example = Example.empty(loader.sdk ?? Sdk.java);
+      _handleLoadException(loader, ex);
+    } on Exception catch (ex) {
+      example = Example.empty(loader.sdk ?? Sdk.java);
+      _handleLoadException(loader, ex);
```

Comment:
Explain why we do not throw in one of the cases.

Reply:
Is this relevant now?